File: python/Disk.py
class Disk:
    type = ""
    capacity = ""
    price = 0

    def __init__(self, type, capacity, price):
        self.type = type
        self.capacity = capacity
        self.price = price

    def setType(self, type):
        self.type = type

    def getType(self):
        return self.type

    def setCapacity(self, capacity):
        self.capacity = capacity

    def getCapacity(self):
        return self.capacity

    def setPrice(self, price):
        self.price = price

    def getPrice(self):
        return self.price

    def showDisk(self):
        print("Disk Type       : " + self.getType())
        print("Disk Capacity   : " + self.getCapacity())
        print("Disk Price      : " + str(self.getPrice()))

File: PHP/Disk.php
<?php
class Disk
{
    private $type;
    private $capacity;
    private $price;

    function __construct($type = "", $capacity = "", $price = 0)
    {
        $this->type = $type;
        $this->capacity = $capacity;
        $this->price = $price;
    }

    function setType($type)
    {
        $this->type = $type;
    }

    function getType()
    {
        return $this->type;
    }

    function setCapacity($capacity)
    {
        $this->capacity = $capacity;
    }

    function getCapacity()
    {
        return $this->capacity;
    }

    function setPrice($price)
    {
        $this->price = $price;
    }

    function getPrice()
    {
        return $this->price;
    }

    function showDisk()
    {
        echo "Disk Type       : " . $this->getType() . "<br>";
        echo "Disk Capacity   : " . $this->getCapacity() . "<br>";
        echo "Disk Price      : " . $this->getPrice() . "<br>";
    }

    function __destruct(){}
}